instrumentation/opentelemetry_oban/lib/opentelemetry_oban/job_handler.ex

```diff
@@ -64,14 +64,13 @@ defmodule OpentelemetryOban.JobHandler do
       Trace.messaging_destination() => queue,
       Trace.messaging_destination_kind() => :queue,
       Trace.messaging_operation() => :process,
-      :"messaging.oban.job_id" => id,
-      :"messaging.oban.worker" => worker,
-      :"messaging.oban.priority" => priority,
-      :"messaging.oban.attempt" => attempt,
-      :"messaging.oban.max_attempts" => max_attempts,
-      :"messaging.oban.inserted_at" =>
-        if(inserted_at, do: DateTime.to_iso8601(inserted_at), else: nil),
-      :"messaging.oban.scheduled_at" => DateTime.to_iso8601(scheduled_at)
+      :"oban.job.job_id" => id,
+      :"oban.job.worker" => worker,
+      :"oban.job.priority" => priority,
+      :"oban.job.attempt" => attempt,
+      :"oban.job.max_attempts" => max_attempts,
+      :"oban.job.inserted_at" => if(inserted_at, do: DateTime.to_iso8601(inserted_at), else: nil),
```

Comment:
`if` already returns `nil` if no `else`
```suggestion
      :"oban.job.inserted_at" => if(inserted_at, do: DateTime.to_iso8601(inserted_at)),
```

Reply:
`Oban` jobs always have the `inserted_at` field
```suggestion
      :"oban.job.inserted_at" => DateTime.to_iso8601(inserted_at),
```

Reply:
What do you want the resolution to be here, @bryannaegele ?

Reply:
So is it the case that all jobs have inserted at and it is populated as @joaothallis says?

Reply:
From Parker:
> Jobs always have an inserted_at once they’re written to the database.

Reply:
Cool. So the last suggestion by Joao works?

Reply:
Ready ✅